src/test/java/net/snowflake/client/RunningNotOnGithubActionsMac.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2012-2019 Snowflake Computing Inc. All right reserved.
+ */
+package net.snowflake.client;
+
+import net.snowflake.client.core.Constants;
+
+public class RunningNotOnGithubActionsMac implements ConditionalIgnoreRule.IgnoreCondition {
+  public boolean isSatisfied() {
+    return isRunningOnGithubActionsMac();
+  }
+
+  public static boolean isRunningOnGithubActionsMac() {
+    return TestUtil.systemGetEnv("GITHUB_ACTIONS") != null && Constants.getOS() == Constants.OS.MAC;
```

Comment:
we don't have MacOS tests on jenkins - do we need to combine mac with GH? beside that we have a separate annotation for not running on GH actions

Reply:
We may need to run it locally on Mac for debugging

Reply:
please create a follow up ticket to investigate

Reply:
done: https://snowflakecomputing.atlassian.net/browse/SNOW-1555950